app/flows/check_uk_visa_flow/outcomes/outcome_work_y_academic.erb

```diff
@@ -0,0 +1,170 @@
+<% text_for :title do %>
+  You need a visa to work as an academic or researcher
+<% end %>
+
+<% govspeak_for :body do %>
+  Based on your answers, you have the following visa options.
+
+  <%= render partial: 'stateless_or_refugee', locals: {calculator: calculator} %>
+
+  ----
+  <% if calculator.passport_country_in_british_national_overseas_list? %>
```

Comment:
If this is the same for each outcome could it be pulled out into a partial? and the same for the Youth Mobility Scheme?

Reply:
Yes it is, good idea Ed! I've pulled both BNO and Youth Mobility Scheme options into their own partials which has DRY'd things up nicely.